### PR TITLE
Add DID URI scheme definition as required output

### LINKED ISSUE #5: DID charter does not include defining DID (scheme) itself
The charter only lists the specification of DID documents, but not the DID url itself (the DID scheme).  This is a critical omission.

### PR DESCRIPTION
**Pull request by burnburn**
Fix #5 .

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,7 +167,7 @@ <h1 id="title">Decentralized Identifier Working Group Charter</h1>
       <p class="mission">
 The <strong>mission</strong> of the
 <a href="https://www.w3.org/2019/did-wg/">Decentralized Identifier Working Group</a>
-is to standardize the data model and syntax of Decentralized Identifier
+is to standardize the DID URL scheme as well as the data model and syntax of Decentralized Identifier
 Documents, which contain information related to DIDs that enable the
 aforementioned use cases.</p>
```
Comment (kimdhamilton) [inline index.html]: TODO (@kimdhamilton): ensure change to aforementioned is preserved

Comment (kimdhamilton) [reply index.html]: TODO (@kimdhamilton): this should be URI

File: index.html [hunk 2]
```diff
@@ -229,6 +229,9 @@ <h2>Scope</h2>
         <p>The Working Group will:</p>
         <ol>
           <li>
+Define the DID URL scheme.
```
Comment (kimdhamilton) [inline index.html]: TODO (@kimdhamilton): this should be URI

**Review by brentzundel — APPROVED** [review]

**Review by msporny — APPROVED** [review]
LGTM.

**Review by ken-ebert — APPROVED** [review]

**Comment by kimdhamilton** [comment]
Leaving out scheme was an oversight. This change is approved.

**Pull request merged**